### PR TITLE
added review: summary_generation_sequences.md

### PR DESCRIPTION
**Pull request by hardmaru**
added review of http://arxiv.org/abs/1308.0850

### CODE REVIEW

**Comment by hardmaru** [comment]
I signed it! (/ am a Googler)

File: magenta/review/summary_generation_sequences.md [hunk 1]
```diff
@@ -59,19 +59,19 @@ The RNN’s task is to model the conditional joint probability distribution of t
...
+The method outlined in the paper is to approximate the conditional distribution of the X and Y as a *mixture gaussian distribution*, where many small gaussian distributions are added together, and S is a Bernoulli random variable. This technique of using a neural network to generate the parameters of a mixture distribution was originally developed by [Bishop](https://www.researchgate.net/publication/40497979_Mixture_density_networks), for feed-forward networks, and this paper extends that approach to RNNs. At each time step, the RNN converts *(x[n], y[n], s[n], h[n])* into the *parameters* of a mixture gaussian PDF, which varies over time as it writes. For example, imagine that the RNN has seen the previous few data points (grey dots), and now it must predict a probability distribution over the location of the next point (the pink regions):
```
Comment (bryongloden) [inline magenta/review/summary_generation_sequences.md]: Greetings @hardmaru Thats nice

File: magenta/review/summary_generation_sequences.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
+---
```
Comment (cghawthorne) [inline magenta/review/summary_generation_sequences.md]: Since this isn't a jekyll blog post, I think you can skip this yaml frontmatter.

**Comment by cghawthorne** [comment]
This was merged in #73

**Pull request closed without merging**